style.css

```diff
@@ -383,3 +393,133 @@ header {
   background: #6070ff;
   color: #fff;
 }
+
+.nav-desktop {
+  display: none;
+}
+
+@media all and (min-width: 768px) {
+  /* ===============================*\
+           Header
+*\=============================== */
+
+  .nav-desktop {
+    display: block;
+  }
+
+  .menu-desktop {
+    list-style: none;
+    display: flex;
+    gap: 20px;
+  }
+
+  header {
+    padding: 0;
+  }
+
+  header .navbar .menu-desktop a {
+    font-size: 15px;
+    line-height: 20px;
+    font-style: normal;
+    font-weight: 500;
+    color: #344563;
+  }
+
+  header .navbar .menu-desktop a:hover {
+    color: #6070ff;
+  }
+
+  .hamb-bu {
+    display: none;
+  }
+
+  .navbar {
+    padding: 15px 142px;
+  }
+
+  /* ===============================*\
+           headline
+*\=============================== */
+
+  #headline {
+    background-image: url("images/Header-Desktop.svg");
+    justify-content: center;
+    padding: 0 110px;
+  }
+
+  #headline h1 {
+    font-size: 48px;
+  }
+
+  #headline p {
+    font-size: 20px;
+  }
+
+  /* ===============================*\
+           Work
+*\=============================== */
+  .portfolio-snapshot {
+    flex-direction: row;
+  }
+
+  #work {
+    padding: 142px;
+    margin: 0;
+  }
+
+  .img {
+    width: 100%;
+    cursor: pointer;
+  }
+
+  .order {
+    order: 2;
+  }
+
+  .work-description {
+    margin-top: 20px;
+  }
+
+  /* ===============================*\
+           About me
+*\=============================== */
+
+  #about-section {
+    padding: 114px 142px;
+    display: flex;
+    gap: 30px;
+  }
+
+  .about {
+    width: 40%;
+    padding: 0;
+  }
+
+  .language {
+    flex-direction: row;
+    width: 60%;
+  }
```

Comment:
![s](https://user-images.githubusercontent.com/93039359/200132403-f2f34f38-a90f-4bd3-bdbc-60c4efbd1b4c.PNG)


- [ ] As the previous reviewer suggested, kindly make all these three language boxes of same sizes to follow the Figma template 😊